### ISSUE TITLE
What is the proper weight for feature-matching loss?

### ISSUE THREAD

**Issue by faradayin** [jik876/hifi-gan#82]
There is a $\lambda_{fm}$ weight for feature-matching loss in the paper, but it is not in the code. 
Is it necessary and what is the proper weight value？

**Comment by Linths** [comment]
Hi Faradayin, I had the same issue. After some digging I found out the weight is [applied inside the loss function](https://github.com/jik876/hifi-gan/blob/4769534d45265d52a904b850da5a622601885777/models.py#L257).

To the makers, I was curious if there is a reason to apply the weight there?
Anyways thanks for the open-source codebase :)

**Comment by Ziyi6** [comment]
@faradayin @Linths Hey guys, did you try different weight values? I plotted the feature mapping loss of mpd and msd while training and found out that these losses are increasing during training. Is it normal? 

<img width="1780" height="927" alt="Image" src="https://github.com/user-attachments/assets/9f7cea67-cfdf-4202-a4a8-a2b6fcdbe123" />
